core-header-panel.html

```diff
@@ -150,7 +150,7 @@
           <content id="mainContent" select="*"></content>
         </div>
 
-        <div id="dropShadow" class-name="[[_dropShadowClass]]"></div>
+        <div id="dropShadow" hidden$="[[_dropShadowVisibility]]"></div>
```

Comment:
You can skip the intermediate property now and put the computed "call" directly in the template:

```html
<div id="dropShadow" hidden$="[[computeDropShadowVisibility(_atTop, mode, shadow)">
```